backend/LeetWars.Builder/Services/TarManagementService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using LeetWars.Builder.Interfaces;
+using SharpCompress.Archives.Tar;
+using SharpCompress.Common;
+using SharpCompress.Writers;
+using SharpCompress.Writers.Tar;
+using System.Text;
+
+namespace LeetWars.Builder.Services
+{
+    public class TarManagementService : ITarManagementService
+    {
+        public byte[] SingleFileToTarBytes(string input, string newFileNameWithExtension)
+        {
+            byte[] tarBytes;
+
+            using (MemoryStream ms = new())
+            {
+                using (TarWriter writer = new(ms, new TarWriterOptions(CompressionType.None, false)))
+                {
+                    using MemoryStream inputMs = new(Encoding.UTF8.GetBytes(input));
+
+                    writer.Write(newFileNameWithExtension, inputMs);
+                }
+
+                tarBytes = ms.ToArray();
+            }
```

Comment:
will be more readable use using without brackets

Reply:
Removed brackets